lib/puppet/type/file.rb

```diff
@@ -83,13 +83,12 @@ def self.title_patterns
         use copy the file in the same directory with that value as the extension
         of the backup. (A value of `true` is a synonym for `.puppet-bak`.)
       * If set to any other string, Puppet will try to back up to a filebucket
-        with that title. See the `filebucket` resource type for more details.
-        (This is the preferred method for backup, since it can be centralized
-        and queried.)
+        with that title. Puppet automatically creates a **local** filebucket
+        named `puppet` if one doesn't already exist.See the `filebucket` resource
```

Comment:
```suggestion
        named `puppet` if one doesn't already exist. See the `filebucket` resource
```
nit: missing space after period